etc/disable-common.inc

```diff
@@ -117,8 +117,12 @@ read-only ${HOME}/.reportbugrc
 read-only ${HOME}/.xmonad
 read-only ${HOME}/.xscreensaver
 
-# The user ~/bin directory can override commands such as ls
+# Make directories commonly found in $PATH read-only
 read-only ${HOME}/bin
+read-only ${HOME}/.gem
+read-only ${HOME}/.local
```

Comment:
I think having .local commonly read-only is a bit restrictive.
A lot of tools store their data in .local/share.